src/material/datepicker/_datepicker-theme.scss

```diff
@@ -158,3 +158,21 @@ $mat-calendar-weekday-table-font-size: 11px !default;
     }
   }
 }
+
+@mixin mat-datepicker-density($density-scale) {}
```

Comment:
We should probably hide the non-experimental density mixins from the public API for now (even though they don't do anything). Not exactly sure _how_ to do that without switching to the new Sass module system, though...

Reply:
I can prefix all those mixins with an underscore?

Reply:
I pushed a follow-up commit to this PR that prefixes those with an underscore. I think that should be sufficient?